scripts/arm32_ci_script.sh

```diff
@@ -9,23 +9,23 @@ function usage {
     echo '    corefx source is at ~/cfx'
```

Comment:
Could you also please move this script under the cross folder like the other similar scripts?